Task1/Program.cs

```diff
@@ -10,7 +10,268 @@ class Program
+            int i = 1;
+            while (i*i<N)
+            {
+                Console.Write(i*i +" ");
+                i++;
+            }
+        }
+
+        public static void ReverseOfNumber()
+        {
+            int number = 13456;
+            int reverseNumber = 0;
+            int remainder = 0;
+            while (number > 0)
+            {
+                remainder = number % 10;
+                reverseNumber = reverseNumber * 10;
+                reverseNumber = reverseNumber + remainder;
+                number = number /10;
+            }
+            Console.WriteLine(reverseNumber);
+        }
+
+        public static void PowOfNumber(int number, int n)
+        {
+            int count = 1;
+            int pow = number;
+            while (count < n)
+            {
+                pow = pow *number ;
+                count++;
+            }
+            Console.WriteLine($"{number}^{n} = {pow}");
+        }
+
+        public static void PropertyOfNumber()
+        {
+            Console.WriteLine("Enter number:");
+            int number = Convert.ToInt32(Console.ReadLine());
+            Console.WriteLine("1. Whether the entered number is positive or negative: ");
+            Console.WriteLine("2. Is it simple");
+            Console.WriteLine("3. Is it divided into 2, 5, 3, 6, 9 without a residue");
+            Console.WriteLine("Choose the action:");
+            int action = Convert.ToInt32(Console.ReadLine());
+            switch (action)
+            {
+                case 1:
+                    if (number < 0)
+                    {
+                        Console.WriteLine(" The number is negative.");
+                    }
+                    else
+                        Console.WriteLine(" The number is positive.");
+                    break;
+                case 2:
+                    int count = 1;
+                    for (int i = 1; i <= 5; i++)
+                    {
+                        if (number % i == 0)
+                            count++;
+                    }
+                    if (count > 2)
+                    {
+                        Console.WriteLine(" The number is not simple.");
+                    }
+                    else
+                        Console.WriteLine(" The number is simple.");
+
+                    break;
+                case 3:
+                    string deviding = "";
+                    if (number % 2 == 0)
+                        deviding = deviding + " 2 ";
+                    if (number % 5 == 0)
+                        deviding = deviding + " 5 ";
+                    if (number % 3 == 0)
+                        deviding = deviding + " 3 ";
+                    if (number % 6 == 0)
+                        deviding = deviding + " 6 ";
+                    if (number % 9 == 0)
+                        deviding = deviding + " 9 ";
+                    //придумать что-то с запятыми!
```

Comment:
to add a comma you can use new knowledge obtained today.
You know that string is array of chars, so you can iterate them and add commas where you want.